whylog_vim/gui/windows.py

```diff
@@ -0,0 +1,95 @@
+import vim
+
+from whylog_vim.consts import WindowTypes
+from whylog_vim.gui.vim_commander import VimCommander
+
+
+class WindowContext(object):
+
+    def __enter__(self):
+        VimCommander.set_modifiable()
+        return VimCommander.get_buffor()
+
+    def __exit__(self, type_, value, traceback):
+        VimCommander.set_nomodifible()
+
+
+class Window(object):
+
+    def __init__(self, name, content, modifiable, size=None):
```

Comment:
'name -> file_name'
It is not a file but only whylog window. This class use only for whylog_windows not vim windows with files. So the better name will be whylog_window_name but it is clear that is whylog_window so for me the name of the variable and attribute 'name' is ok.